http/handler.go

```diff
@@ -31,6 +28,7 @@ const (
 // All other requests are passed down to the sub handler.
 type Handler struct {
 	name string
+	r    chi.Router
 	// MetricsHandler handles metrics requests
 	MetricsHandler http.Handler
 	// ReadyHandler handles readiness checks
```

Comment:
the DebugHandler field is never being set anywhere. I'm trying to find anything that uses it but I can't find anything. Can we axe the debug handler since its not being used?

https://github.com/influxdata/influxdb/pull/16451/files